packages/scripts/src/getConversationText.ts

```diff
@@ -0,0 +1,41 @@
+import { MongoClient, BSON } from "mongodb";
+import { assertEnvVars } from "mongodb-rag-core";
+import { Conversation } from "mongodb-chatbot-server";
+
+import "dotenv/config";
+
+const { MONGODB_DATABASE_NAME, MONGODB_CONNECTION_URI } = assertEnvVars({
+  MONGODB_DATABASE_NAME: "",
+  MONGODB_CONNECTION_URI: "",
+});
+
+const conversationIdString = process.argv[2];
+if (!conversationIdString) {
+  throw new Error(
+    "Missing conversation id. Usage: getConversationText <conversationId>"
```

Comment:
nit: maybe mention the npm run command as well b/c that's prob the only way this script is run for now. 

eg:

```sh

```
```suggestion
    "Missing conversation id. Usage: getConversationText <conversationId> (or via package.json: npm run getConversationText -- <conversationId>)"
```